lep/DRAFT/012_Pro_version_personal_server.md

```diff
@@ -0,0 +1,142 @@
+# LEP 012 - Pro version, personal server
+
+Date: September 9th, 2015
+
+Status: Draft
+
+Author: uaalto
+
+## Abstract
+
+A proposal for a Lantern Pro version consisting of a premium personal server, automatically launched after payment has been processed.  The launched server and the requesting client are automatically configured.  Eventually, these personal servers should be shareable at will by the user to other Lantern instances or users.
+
+## Rationale
+
+This LEP stems from the need to accomodate more users within reasonable costs, focusing in one of the different solutions to the problem: diverting users to personal customer-supported servers.  The servers could be shared by more than one user, as long as the user agrees and takes an active decision to do so.  This sharing should be easy in order to facilitate the mechanism and encourage diverse growth approaches for Lantern.
+
+
+## Proposal
+
+### 1. Payment UI
+
+The process of requesting a personal server should be initiated from the Lantern UI.  This sketch proposes a simple way of including it in current Lantern interface.  This can be revisited from the UX/UI perspective, but it serves as a solid starting point.
+
+<p align="center">
+<img src="images/lep-012-01.png" alt="UI Proposal"/>
+</p>
+
+Since June 2014, Stripe can accept Alipay payments, which opens the door to offering this service in China.  Using Alipay with Stripe is straightforward, and just requires an option in the HTML5 data fields.  The default UI is as follows:
```

Comment:
Maybe superfluous, but we'll accept credit card as an always available option right?